internal/scheduler/scheduler.go

```diff
@@ -487,6 +558,20 @@ func (s *Scheduler) generateUpdateMessagesFromJob(job *jobdb.Job, jobRunErrors m
 		} else if lastRun.Failed() {
 			job = job.WithFailed(true).WithQueued(false)
 			runError := jobRunErrors[lastRun.Id()]
+			if lastRun.Returned() {
+				errorMessage := fmt.Sprintf("Maximum number of attempts (%d) reached - this job will no longer be retried", s.maxAttemptedRuns)
+				if job.NumAttempts() < s.maxAttemptedRuns {
+					errorMessage = fmt.Sprintf("Job was attempeted %d times, and has been tried once on all nodes it can run on - this job will no longer be retried", job.NumAttempts())
```

Comment:
```suggestion
					errorMessage = fmt.Sprintf("Job was attempted %d times, and has been tried once on all nodes it can run on - this job will no longer be retried", job.NumAttempts())
```